www/docs/adapters/prisma.md

```diff
@@ -14,7 +14,7 @@ You may have noticed there is a `prisma` and `prisma-legacy` adapter. This is du
 To use this Adapter, you need to install Prisma Client, Prisma CLI, and the separate `@next-auth/prisma-adapter` package:
 
 ```
-npm install @prisma/client @next-auth/prisma-adapter
+npm install --save @prisma/client @next-auth/prisma-adapter
```

Comment:
This is the default in `npm`, so I don't think this should be necessary:

https://docs.npmjs.com/cli/v7/commands/npm-install#save